agency/README.md

```diff
@@ -203,6 +216,14 @@ Body Params:
 | `missing_param`     | A required parameter is missing | Array of missing parameters     |
 | `unregistered`      | Vehicle is not registered       |                                 |
 
+### Trip_id Requirements
+
+#### Micromobility
```

Comment:
Interesting to include each mode here, which makes sense, but wondering instead how this could go in the appropriate /mode readme instead and linked to.